.editorconfig

```diff
@@ -12,7 +12,7 @@ indent_size = 4
 [*.ts]
 # This is being picked up by prettier/eslint, so in order to leave the existing
 # code alone for now, we overwrite max_line_length for TypeScript code.
-max_line_length = 100
+max_line_length = 80
```

Comment:
I'd argue you can get rid of the whole `[*.ts]` section now - `max_line_length` is the only thing we were setting, and it's already being set to `80` above. The comment above this line also doesn't make sense anymore. So I think lines 12-16 can just go.